launcher/container_runner.go

```diff
@@ -129,8 +129,15 @@ func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.To
 		return nil, err
 	}
 
-	if err := enableMonitoring(launchSpec.MonitoringEnabled, logger); err != nil {
-		return nil, err
+	if launchSpec.MonitoringEnabled == spec.All && !launchSpec.Experiments.EnableHealthMonitoring {
```

Comment:
This is a new flag `EnableHealthMonitoring`? It it part of the experiment binary yet?

Reply:
Yes, I had Josh build a new binary this morning and am working on testing it.